hooks/pre-commit

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+
+import commands
+import os
+import sys
+import urllib.request, urllib.parse, urllib.error
+from distutils.version import LooseVersion
+import re
+
+pattern = re.compile(r'script_version["\s:=]+(\d+\.\d+)')
+version_dict = {}
+versionfile = urllib.request.urlopen(
+    "https://raw.githubusercontent.com/henrykironde/retriever/1f18008838cecf7f8ac208d57cd2a578e50e1c2c/version.txt")
```

Comment:
This should be the https://raw.githubusercontent.com/weecology/retriever/master/version.txt